jenkins/build.sh

```diff
@@ -73,26 +74,6 @@ case $LOG_LEVEL in
 esac
 echo "Using VERSION=${VERSION} GIT_VER=${GIT_VER} PRODUCT=${PRODUCT} TEST=${TEST} LOG_LEVEL=${LOG_LEVEL} "
 
-################### INSTALL REQUIRED RPM PACKAGES ##############################
-
-# Check python package
-req_file=$BASE_DIR/low-level/requirements.txt
-echo "Installing python packages..."
-python3 -m pip install -r "$req_file"  > /dev/null || {
```

Comment:
We don't install packages here? You can verify by trying build.sh in a fresh VM.

Reply:
I have tested this code on fresh vm and everything is working as expected.

Reply:
@sandeepsgt we don't required this packages at the time of building rpm's.
Unnecessary this code is installing this pakages.

Reply:
Ok. I think pip installation is not needed now. Earlier incase of pyinstaller, this was needed because it need dependencies to be preinstalled before build to bundle in binary.